lib/util.js

```diff
@@ -205,6 +205,22 @@ function pad(n) {
  */
 function styleText(format, text) {
   validateString(text, 'text');
+
+  if (ArrayIsArray(format)) {
```

Comment:
I think you also need to change this to string | string[]
`/**
 * @param {string} format
 */
 `